arch-install.bash

```diff
@@ -51,6 +51,10 @@ block_device=$(
 
 umount --quiet "$block_device" || true
 
+clear
+echo "Make sure your device is unfrozen if you choose to encrypt your drive!"
```

Comment:
This is only the case if an SSD is being used. It would make sense to clarify that in this warning message.